lua/framework/mesh/unpartitioned_mesh/unpartition_mesh_lua_utils.h

```diff
@@ -195,20 +195,20 @@ int chiUnpartitionedMeshFromMshFormat(lua_State* L);
  * ### Example
  * An example mesh creation below:
  * \code
- * chiMeshHandlerCreate()
+ * MeshHandlerCreate()
  *
- * umesh = chiUnpartitionedMeshFromExodusII("resources/TestObjects/Mesh.e")
+ * umesh = UnpartitionedMeshFromExodusII("resources/TestObjects/Mesh.e")
  *
- * chiSurfaceMesherCreate(SURFACEMESHER_PREDEFINED)
- * chiVolumeMesherCreate(VOLUMEMESHER_UNPARTITIONED, umesh)
+ * SurfaceMesherCreate(SURFACEMESHER_PREDEFINED)
+ * VolumeMesherCreate(VOLUMEMESHER_UNPARTITIONED, umesh)
  *
- * chiSurfaceMesherExecute()
- * chiVolumeMesherExecute()
+ * SurfaceMesherExecute()
```

Comment:
I removed the SurfaceMesher. I guess I didn't a good enough job ...

Reply:
I think we can easily make some more clean-up actions when working on #17.